internal/cron/README.md

```diff
@@ -0,0 +1,125 @@
+[![GoDoc](http://godoc.org/github.com/robfig/cron?status.png)](http://godoc.org/github.com/robfig/cron)
```

Comment:
Can you update this file and explain why it's here? Don't think we need the full README. For example:

```
# cron

This package contains a fork of [`github.com/robfig/cron/v3@v3.0.0`](https://github.com/robfig/cron/) which adds support for mocking the time.

See LICENSE for the license of the original package
```